src/main/java/org/openrewrite/kotlin/tree/K.java

```diff
@@ -1418,7 +1418,17 @@ public List<TypeParameter> getTypeParameters() {
 
         boolean isSetterFirst;
 
-        public Property(UUID id, Space prefix, Markers markers, @Nullable JContainer<TypeParameter> typeParameters, VariableDeclarations variableDeclarations, @Nullable J.MethodDeclaration getter, @Nullable J.MethodDeclaration setter, boolean isSetterFirst) {
+        @Nullable
+        JRightPadded<J> receiverName;
+
+        @Nullable
+        public J getReceiverName() {
```

Comment:
I suggest we ho with `getReceiver()` instead. Since this is a new property I think it would have been nice with a more specific type. Although `J.Identifier` and `J.FieldAccess` may unfortunately not have anything in common except for `J` and `Expression`.